recipes/pyre/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+util-linux
```

Comment:
What is missing from conda-forge that this is needed?

Reply:
pyre captures number of cpus / cores etc. It uses "lscpu" internally for this. Are there officially recommended alternatives?

Reply:
Well, typically we build our dependencies as part of conda-forge.  I recommend putting in a PR for util-linux first. (repo at https://github.com/karelzak/util-linux). Without this dep package, users may install pyre and then not be able to use it because of something missing on the system.  We only allow this for extreme exceptions, such as drivers that need kernel access. This doesn't seem to fit that case.

Reply:
Agreed. There is also a misleadingly named conda package called util-linux that seems to supply only libuuid. Would this be the package that needs updates? 
https://anaconda.org/anaconda/util-linux

Reply:
You should make a PR to add a new package called `util-linux`. It looks like the Anaconda package isn't being used so we don't need to worry about it conflicting.

Reply:
Alright. Will close this PR and get back to this when ready with all the dependencies.